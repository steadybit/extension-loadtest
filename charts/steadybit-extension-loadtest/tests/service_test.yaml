templates:
  - service.yaml
tests:
  - it: manifest should match snapshot
    set:
      loadtest:
        servicesEnabled: "true"
        servicePortFrom: 9000
        servicePortTo: 9002
    asserts:
      - matchSnapshot: {}

