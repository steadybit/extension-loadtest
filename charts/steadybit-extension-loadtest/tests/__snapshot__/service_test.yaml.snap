manifest should match snapshot:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      annotations:
        meta.helm.sh/release-name: steadybit-extension-loadtest-9000
        meta.helm.sh/release-namespace: steadybit-agent
        steadybit.com/extension-auto-discovery: |
          {"extensions":[{"port":9000,"protocol":"http","types":["ACTION","DISCOVERY","EVENT"]}]}
      name: steadybit-extension-loadtest-9000
      namespace: steadybit-agent
    spec:
      ports:
        - port: 9000
          protocol: TCP
          targetPort: 8082
      selector:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: steadybit-extension-loadtest
  2: |
    apiVersion: v1
    kind: Service
    metadata:
      annotations:
        meta.helm.sh/release-name: steadybit-extension-loadtest-9001
        meta.helm.sh/release-namespace: steadybit-agent
        steadybit.com/extension-auto-discovery: |
          {"extensions":[{"port":9001,"protocol":"http","types":["ACTION","DISCOVERY","EVENT"]}]}
      name: steadybit-extension-loadtest-9001
      namespace: steadybit-agent
    spec:
      ports:
        - port: 9001
          protocol: TCP
          targetPort: 8082
      selector:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: steadybit-extension-loadtest
  3: |
    apiVersion: v1
    kind: Service
    metadata:
      annotations:
        meta.helm.sh/release-name: steadybit-extension-loadtest-9002
        meta.helm.sh/release-namespace: steadybit-agent
        steadybit.com/extension-auto-discovery: |
          {"extensions":[{"port":9002,"protocol":"http","types":["ACTION","DISCOVERY","EVENT"]}]}
      name: steadybit-extension-loadtest-9002
      namespace: steadybit-agent
    spec:
      ports:
        - port: 9002
          protocol: TCP
          targetPort: 8082
      selector:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: steadybit-extension-loadtest
